hack/functest.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+DEFAULT_OPERATOR_IMAGE_NAME="cincinnati-operator"
+OPERATOR_IMAGE_NAME="${OPERATOR_IMAGE_NAME:-${DEFAULT_OPERATOR_IMAGE_NAME}}"
+
+# Override the image name when this is invoked from openshift ci
+if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
+	CATALOG_FULL_IMAGE_NAME="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:${OPERATOR_IMAGE_NAME}"
+	echo "Openshift CI detected, deploying using image $CATALOG_FULL_IMAGE_NAME"
+fi
+
+GOBIN="${GOBIN:-$GOPATH/bin}"
+GINKGO=$GOBIN/ginkgo
+
+if ! [ -x "$GINKGO" ]; then
+	echo "Retrieving ginkgo and gomega build dependencies"
+	go get github.com/onsi/ginkgo/ginkgo
+	go get github.com/onsi/gomega/...
+else
+	echo "GINKO binary found at $GINKGO"
+fi
+
+
+"$GOBIN"/ginkgo build functests/
+# Run functional tests
+"$GOBIN"/ginkgo functests
+
+if [ $? -ne 0 ]; then
+	hack/dump-debug-info.sh
+	echo "ERROR: Functest failed."
+	exit 1
+fi
```

Comment:
I don't think we don't need this, at least not yet.

Reply:
I like to keep it for now and remove later if needed in the next PR when more changes/new code is put together.

Reply:
You won't need it. I did that for a very specific reason in the other project.

Reply:
ok, removed it.